modules/claims_api/app/controllers/claims_api/v2/veterans/power_of_attorney_controller.rb

```diff
@@ -95,28 +97,35 @@ def submit_power_of_attorney(poa_code, form_number)
           ClaimsApi::PoaFormBuilderJob.perform_async(power_of_attorney.id, form_number)
 
           render json: ClaimsApi::V2::Blueprints::PowerOfAttorneyBlueprint.render(
-            representative(poa_code).merge({ id: power_of_attorney.id, code: poa_code }),
+            format_representative(representative(poa_code)).merge({ id: power_of_attorney.id, code: poa_code }),
             root: :data
           ), status: :accepted, location: url_for(
             controller: 'power_of_attorney', action: 'show', id: power_of_attorney.id
           )
         end
 
         def representative(poa_code)
-          organization = ::Veteran::Service::Organization.find_by(poa: poa_code)
-          return format_organization(organization) if organization.present?
-
           individuals = ::Veteran::Service::Representative.where('? = ANY(poa_codes)',
                                                                  poa_code).order(created_at: :desc)
-          return {} if individuals.blank?
 
-          if individuals.pluck(:representative_id).uniq.count > 1
+          if individuals.empty?
+            raise ::Common::Exceptions::ResourceNotFound.new(
+              detail: "Could not retrieve Power of Attorney with poa code: #{poa_code}"
+            )
+          end
+
+          if (individuals&.pluck(:representative_id)&.uniq || []).count > 1
```

Comment:
do we need the ` || [] ` here? doesn't `individuals` return empty and the line above raises an error?

even if it makes it to this line, we would get an empty array and a count of 0.
```
3.2.2 :003 > [].pluck(:representative_id).uniq
 => []
3.2.2 :004 > [].pluck(:representative_id).uniq.count
 => 0
```